packages/api/src/DataBuilder.ts

```diff
@@ -7,10 +7,12 @@ import {SEVEN_DAYS_AGO} from './constants.js'
 export type BlueAsset = {
 	id: number
 	name: string
+	rank: number
 	symbol: string
 	logo: string | null
 	website: string | null
 	repos: string[]
+	twentyFourHourPriceChange: number
```

Comment:
the heck is that prop name? xD can't you rename to `change24h`, numbers better for quick catch

Reply:
`asset['change24h']` or `asset.change24h`